.circleci/config.yml

```diff
@@ -378,7 +378,10 @@ jobs:
       - node/install:
           install-yarn: true
       - run:
-          command: yarn && yarn build && yarn apitests
+          command: yarn && yarn build && yarn extract-api
+      - run:
+          name: Check if API has changed
+          command: git diff --exit-code api-report/**
```

Comment:
The idea is to have the typescript files act as a kind of "snapshot" test, so if it changes, we need to manually update them by running `yarn extract-api` and commiting the difference.